apps/content/app/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ |  Copyright 2020 Adobe Systems Incorporated
+ |
+ |  Licensed under the Apache License, Version 2.0 (the "License");
+ |  you may not use this file except in compliance with the License.
+ |  You may obtain a copy of the License at
+ |
+ |      http://www.apache.org/licenses/LICENSE-2.0
+ |
+ |  Unless required by applicable law or agreed to in writing, software
+ |  distributed under the License is distributed on an "AS IS" BASIS,
+ |  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ |  See the License for the specific language governing permissions and
+ |  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.adobe.dx</groupId>
+        <artifactId>content.reactor</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>content-apps</artifactId>
+    <packaging>content-package</packaging>
+
+    <name>${module.name} - App Package</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>filevault-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <packageType>application</packageType>
+                    <properties>
+                        <cloudManagerTarget>none</cloudManagerTarget>
+                    </properties>
+                    <filters>
+                        <filter><root>/apps/dx/components/content/marketo</root></filter>
+                    </filters>
+                    <embeddeds>
+                        <embedded>
+                            <groupId>com.adobe.dx</groupId>
+                            <artifactId>com.adobe.dx.content.core</artifactId>
+                            <target>/apps/dx/install</target>
+                            <filter>true</filter>
+                        </embedded>
+                        <embedded>
+                            <groupId>com.adobe.dx</groupId>
+                            <artifactId>content-content</artifactId>
```

Comment:
Note that this `content` in the sense of Skyline - This package would hold only non-`apps/libs` nodes.

Reply:
erh...content-content sounds weird

Reply:
Agreed. But not sure what else to call it. :(

Reply:
ok, content-contnet sounds weird, but ok